public/includes/find-house.php

```diff
@@ -0,0 +1,65 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sqlQuery = "SELECT * FROM `states` WHERE 1";
```

Comment:
Can we use `snake case` for naming PHP variables and stick to `camel case` for Javascript? I noticed that you missed the naming type in this file.